classes/profile.php

```diff
@@ -201,8 +201,8 @@ public static function save(\ExcimerLog $log, int $reason, int $created, float $
             'duration' => $duration,
             'request' => $_SERVER['PHP_SELF'] ?? 'UNKNOWN',
             'parameters' => $parameters,
-            'cookies' => !defined(NO_MOODLE_COOKIES),
-            'buffering' => !defined(NO_OUTPUT_BUFFERING),
+            'cookies' => !defined('NO_MOODLE_COOKIES') || !NO_MOODLE_COOKIES,
```

Comment:
isn't this boolean reversed?

ie if cookies = 1 should that mean we did have a cookie?